entities.csv

```diff
@@ -383,6 +385,7 @@ ITGA2B3
 ITGB
 ITPR
 JAK
+JAMM

```

Comment:
Maybe `JAMM_complex?`

Reply:
@lauradohertyws said that the JAMMs are a functional classification, and it's not that they all form an active complex together. She also didn't have anything in mind on a sub-hierarchy within JAMM

Reply:
Makes sense